src/Table/index.jsx

```diff
@@ -93,10 +93,13 @@ class Table extends React.Component {
       <thead
         className={classNames(...this.props.headingClassName.map(className => styles[className]))}
       >
-        <tr>
+        <tr className={classNames({ 'd-flex': this.props.tableFixed })}>
           {this.props.columns.map(col => (
```

Comment:
If you want to get even more terse you could consider doing

```javascript
{ this.props.columns.map(({ key, columnSortable, tableFixed, width }) => ... }
```

so it doesn't have to be `col.____` inside the mapping function.

Reply:
I'd do that, but I need a reference to `col` a few lines down inside the map function to pass it to `this.getTableHeading`.

Reply:
Oh I didn't see that - good call!

I don't think it's useful here then.

Maybe more relevant for [`line 119`](https://github.com/edx/paragon/pull/179/files#diff-f387f1e6539c577f164d015dc0c5f453R119)?

Either way, it's a nit and doesn't really matter.

Reply:
I can do that in another map below this though. :+1: